user.go

```diff
@@ -74,8 +75,14 @@ type User struct {
 
 	// ProfileURI is a pointer to where their profile picture lives
 	ProfileURI string `bson:"profileUri" json:"profileUri,omitempty" xml:"profileUri,omitempty"`
+
+	// mfaFactors contains the MFA factors the user is signed up to
+	MfaFactors map[string]MultiFactorType `bson:"MultiFactorType" json:"MultiFactorType,omitempty" xml:"MultiFactorType,omitempty"`
 }
 
+// MultiFactorType specifies the types of authentication a user can enrol in;
+type MultiFactorType int32
```

Comment:
Looks like you've added the beginning of an enum type here, but are missing the expected types i.e. Authenticator, Email and SMS?